src/com/elixr/training/Main.java

```diff
@@ -3,44 +3,60 @@
 import java.io.File;
 import java.nio.file.Files;
 import java.nio.file.Paths;
-import java.util.StringTokenizer;
 import java.lang.String;
+import java.sql.*;
+
+import com.elixrdb.training.JdbcConnection;
 
 public class Main {
-    static  final String TXTFILE = ".txt" ;
-    static final String JSONFILE = ".json";
-    public static void main(String[] args) {
+    static JdbcConnection database = new JdbcConnection();
+    static int count = 0;
+    static String inputFilePath;
+    static String searchWord;
 
+    public static void main(String[] args) {
         if (args.length != 2) {
-            System.out.println(" File path and word to search are missing. Exiting the program... ");
+            System.out.println(Constants.ERROR_MESSAGE_ARGUMENTS_NOT_FOUND);
+            try {
+                database.dbOperation(inputFilePath, searchWord, Constants.STATUS_FAILURE, count, Constants.ERROR_MESSAGE_ARGUMENTS_NOT_FOUND);
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
             return;
         }
-        String inputFilePath = args[0];
-        String searchWord = args[1];
+        inputFilePath = args[0];
+        searchWord = args[1];
         File file = new File(inputFilePath);
 
         System.out.println("Processing................");
 
         if (!file.exists()) {
-            System.out.println("file doesn't exists");
+            System.out.println(Constants.ERROR_MESSAGE_FILE_NOT_EXISTS);
```

Comment:
Missing database entries for these error. There are others also missing, please update all of them